### PR TITLE
[RFR][1LP][NOTEST] Remove all references to psphere

### PR DESCRIPTION
**Pull request by rsnyman**
We no longer use psphere, wrapanapi moved to pyVmomi. Remove psphere from the requirements, and use the methods in wrapanapi to delete the VirtualCentre vms and templates.

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by rsnyman** [comment]
Just ran the test for fun, even though I didn't change any of the code. I'm not sure why 5.7 is failing, but it's not related to my changes.

File: scripts/delete_vm.py [hunk 1]
```diff
@@ -275,17 +272,16 @@ def vsphere_check_with_api(api, run_time, text):
...
     templates_to_delete = []
 
-    nightly_templates = [VirtualMachine.get(api.api, name=x)
-                         for x in api.list_template() if "miq-nightly" in x]
-    nightly_templates.sort(key=lambda x: datetime.datetime.strptime(x.name[-12:], "%Y%m%d%H%M"))
+    nightly_templates = filter(lambda t: 'miq-nightly' in t, api.list_template())
+    nightly_templates.sort(key=lambda x: datetime.datetime.strptime(x[-12:], '%Y%m%d%H%M'))
 
     if len(nightly_templates) > MRU_NIGHTLIES:
         for template in nightly_templates[:-MRU_NIGHTLIES]:
```
Comment (RonnyPfannschmidt) [inline scripts/delete_vm.py]: based on
```
>>> a = [1,2,3]
>>> a[:-1]
[1, 2]
>>> a[:-4]
[]
>>> 
```

i propose `templates_to_delete = nightly_templates[:-MRU_NIGHTLIES]` - neither the if not the loop are needed

File: scripts/delete_vm.py [hunk 2]
```diff
@@ -89,8 +88,7 @@ def rhevm_vm_status(api, vm_name):
 
 
 def vsphere_vm_status(api, vm_name):
-    vm = VirtualMachine.get(api.api, name=vm_name)
-    return vm.runtime.powerState
+    return api.vm_status(vm_name)
 
 
 def rhevm_is_vm_up(api, vm_name):
```
Comment (RonnyPfannschmidt) [inline scripts/delete_vm.py]: **QUESTION**

can we replace all of those with the is_vm_running/is_vm_stopped wrapanapic alls (i just took a look and noted the bascia apis do this more correct)

**Pull request merged**